C/common/reading_circularbuffer.cpp

```diff
@@ -0,0 +1,96 @@
+/*
+ * Fledge reading circular buffer class
+ *
+ * Copyright (c) 2025 Dianomic Systems
+ *
+ * Released under the Apache 2.0 Licence
+ *
+ * Author: Mark Riddoch
+ */
+#include <reading_circularbuffer.h>
+
+using namespace std;
+
+/**
+ * Create a circular buffer of readings
+ *
+ * @param size	The numebr of items to retain oin the circular buffer
```

Comment:
The numebr of items to retain oin the circular buffer -> The number of items to retain in the circular buffer